walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,67 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+  font-family: Arial, sans-serif;
+  padding: 4px;
+  text-align: center;
 }
 
-.image-thumbnails {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+h1 {
+  text-align: center;
+  font-weight: normal;
+}
+
+nav {
+  background: white;
+  width: 100%;
+  margin-bottom: 20px;
+  position: fixed;
+  top: 0;
+  padding-top: 1vw;
+  padding-bottom: 1vw;
+}
+
+nav ul {
+  margin: 0;
+  padding: 0;
+}
+
+nav ul li {
+  list-style: none;
+  text-align: center;
+  font-size: calc(14px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  line-height: calc(1.3em + (1.5 - 1.2) * ((100vw - 300px)/(1600 - 300)));
```

Comment:
Woah... This is some crazy math for a font. Consider just using media breakpoints for the same reasons listed above on `vw`.